src/main/java/edu/uc/cech/agrawadv/eventorganizer/user/UserService.java

```diff
@@ -24,5 +27,5 @@ static void saveUser(User user) {
     	 //TODO
     }
      
-     public List<User> findAll();
+     public List<User> findAllUsers();

```

Comment:
Again just to be more clear on naming variables.
ref.
https://dev.to/khalalw/the-5-commandments-of-naming-variables-1l04